### ISSUE TITLE
Setting up of multicast network in IBMC

### ISSUE THREAD

**Issue by mpaulgreen** [RHEcosystemAppEng/amq-cob#11]
@ccustine has raised a support case with IBM for the above mentioned issue.

To move forward. @rhkp can start with mirroring for the time to set up DR.

**Comment by mpaulgreen** [comment]
@rhkp @sgahlot @ccustine I tried with this example pointed from Red Hat documentation on a single VM and clustering is working. 
https://github.com/apache/activemq-artemis/tree/main/examples/features/clustered/clustered-queue

You can find the reference of the configuration here. 

https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q3/html-single/configuring_amq_broker/index#creating-broker-cluster-with-jgroups-based-dynamic-discovery-configuring


This will enable us to still perform dynamic discovery. Let me know if you have any queries.

**Comment by sgahlot** [comment]
@mpaulgreen how is this different than what we've done so far where we tested with clustering between different brokers on a single VM?

**Comment by mpaulgreen** [comment]
@sgahlot @rhkp @ccustine I updated the link to example. I think I pasted an incorrect link earlier

**Comment by mpaulgreen** [comment]
Reference - https://lists.jboss.org/pipermail/jboss-jira/2017-April/311691.html
@rhkp , can you try with following changes for muti-vm
   - make TCP loopback="false", bind_addr="internal ip of vm1"
   - bind_port=7800
   -corresponding changes Second vm.

Also change all tcp://0.0.0.0 with internal tp://<<internal-ip>> 
the port of second vm 61617 for both connector and acceptor

The warning should be gone. 

Now test the clustering... with both warning and without warning.

FYI @sgahlot and @ccustine

**Comment by rhkp** [comment]
@mpaulgreen 
Thanks. With the inputs I made the changes in jgroup ping file and still was seeing the WARN. 

I then went back to broker.xml and deleted all out of the box extra configs which was present and only kept the configs in the example at the original [sample](https://github.com/apache/activemq-artemis/blob/main/examples/features/clustered/clustered-jgroups/src/main/resources/activemq/server0/broker.xml)  and voila everything is working now.

Thanks again for your guidance and direction. Great weekend ahead.
